src/components/MoneyRequestAmountInput.tsx

```diff
@@ -82,6 +82,9 @@ type MoneyRequestAmountInputProps = {
      * Autogrow input container length based on the entered text.
      */
     autoGrow?: boolean;
+
+    /** Flag indicating if the reset button was clicked to reset the amount */
+    resetClicked?: boolean;
```

Comment:
I think we should improve the naming to be more self-explanatory
Perhaps `shouldResetAmount` would work?